<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>880</width>
    <height>730</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>880</width>
    <height>730</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">ArduGUIno 0.4</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolBox" name="Tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>881</width>
      <height>611</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>881</width>
       <height>543</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Debug Console</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>851</width>
        <height>541</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Received</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>HEX Value</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QPlainTextEdit" name="plainTextEdit_input">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPlainTextEdit" name="plainTextEdit_input_hex">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="plainText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="5">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Sent</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="5">
        <widget class="QPlainTextEdit" name="plainTextEdit_debug">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_Data"/>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="pushButton_Send">
         <property name="text">
          <string>Send</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>881</width>
       <height>543</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
      </font>
     </property>
     <attribute name="label">
      <string>Scan</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>19</y>
        <width>841</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCustomPlot" name="customPlot" native="true">
         <property name="minimumSize">
          <size>
           <width>811</width>
           <height>311</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>168</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="calibration_PB">
           <property name="text">
            <string>Calibrate</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="4">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QSpinBox" name="line_length_spinBox">
             <property name="accelerated">
              <bool>true</bool>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>256</number>
             </property>
             <property name="value">
              <number>256</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Line Length</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QSpinBox" name="sample_size_spinBox_2">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Sample Size</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QSpinBox" name="step_size_spinBox_3">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Set Step Size</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="setup_pushButton">
           <property name="text">
            <string>Setup</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="LoadScan">
           <property name="text">
            <string>Load Scan</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="scanPB">
           <property name="text">
            <string>Start Scan</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>879</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>.SF NS Text</family>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Device:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_deviceSignature">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="baud_label">
       <property name="text">
        <string>Baud Rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="baudDropDown">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="maxCount">
        <number>2147483645</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
       <property name="minimumContentsLength">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>1200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>230400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>250000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000000</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Serial port:</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="serialPortDropdown"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_connect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>sendData()</slot>
 </slots>
</ui>
